packages/@aws-cdk-testing/framework-integ/test/aws-dynamodb/test/integ.import-source.js.snapshot/DynamoImportSourceIntegTestDefaultTestDeployAssert540810E4.template.json

```diff
@@ -95,20 +95,12 @@
   "SingletonFunction1488541a7b23466481b69b4408076b81HandlerCD40AE9F": {
    "Type": "AWS::Lambda::Function",
    "Properties": {
-    "Runtime": {
-     "Fn::FindInMap": [
-      "LatestNodeRuntimeMap",
-      {
-       "Ref": "AWS::Region"
-      },
-      "value"
-     ]
-    },
+    "Runtime": "nodejs18.x",
```

Comment:
I believe you need to rerun the integration test cases after updating the aws-cdk code locally, as I think this is an old way to set the Lambda function used for custom resources

Reply:
Verified that this is the correct behaviour as the integration test does not set context value for feature flag `@aws-cdk/core:target-partitions` to true, thus it should use the old behaviour which is to use nodejs18.

Reply:
curious, if this flag is set to false, how was it modified to region map in the first place, since it seems like we're reverting the current expected behaviour

Reply:
eg. referring the recent PR from Kendra here https://github.com/aws/aws-cdk/pull/30737/files#diff-c4a734742f971b5f2345c1770a91bb8e88fcadc7b5e317c2f88f32082535fbf8